### DISCUSSION TITLE
Error executing process : GET PREDICTION VERSION

### DISCUSSION THREAD

**Discussion by kuaayush** [nextflow-io/nextflow#4341] [Q&A]
The error message is :

Error executing process > 'NFCORE_EPITOPEPREDICTION:EPITOPEPREDICTION:GET_PREDICTION_VERSIONS'
Caused by:
  Process requirement exceeds available memory -- req: 12 GB; avail: 8 GB

I even used the command: _**max_memory '16.GB'**_ after the error but it still shows same error. 
<img width="703" alt="Screenshot 2023-09-26 at 11 16 45 AM" src="https://github.com/nextflow-io/nextflow/assets/81306944/889ea40c-2335-4af1-ace3-582a49a56259">
<img width="1154" alt="Screenshot 2023-09-26 at 3 12 45 PM" src="https://github.com/nextflow-io/nextflow/assets/81306944/26e3d010-d931-469f-aec7-57706016e1c7">

**Comment by bentsherman** [comment]
The error says that your system only has 8 GB of memory available, so you should actually set max_memory to 8 GB.
